react_native/trimblemaps-mobile-sdk-demo-reactnative/react_native/RNMapsSampleApp/ios/get_version.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
```

Comment:
This is our internal versioning file? It probably doesn't make sense to include it in the public project?